util/cloudsqlutil/options.go

```diff
@@ -77,10 +79,18 @@ func WithIAMAccountEmail(email string) Option {
 	}
 }
 
+// WithUserAgent returns an Option that sets the User-Agent.
+func WithUserAgent(ua string) Option {
+	return func(p *engineConfig) {
+		p.userAgents = ua
+	}
+}
+
```

Comment:
I don't think we need this option -- it will overwrite the default agent (which we want to send all the time)